packages/node_modules/@node-red/util/lib/util.js

```diff
@@ -527,68 +526,35 @@ function setObjectProperty(msg,prop,value,createMissing) {
     return true;
 }
 
-/**
+/*!
```

Comment:
```suggestion
/**
```

`/*!` doesnt provide type hints to editor